getline.ts/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
     "module": "commonjs",
+    "lib": ["es2016", "dom"],
```

Comment:
@q3k change the target to es2016 to include it's libs instead of mix-and-matching targets and libs